gaia-header.js

```diff
@@ -91,6 +91,11 @@ module.exports = component.register('gaia-header', {
       titles: this.getElementsByTagName('h1')
     };
 
+    // The [ignoreDir] attribute can be used to force the older behavior of
+    // gaia-header, where the whole header is always displayed in LTR mode.
+    this.ignoreDir = this.hasAttribute('ignoreDir');
+    this.dirObserver &= !this.ignoreDir;
```

Comment:
`dirObserver` is used in gaia-component, where it is used to add `dir` attributes to shadow DOM children whenever the direction of the document changes.

That’s the way it worked before — it was just called `rtl`, I’ve renamed it to a (hopefully) more self-explanatory name.